cfme/utils/log.py

```diff
@@ -221,11 +221,11 @@ def process(self, message, kwargs):
 
 def _load_conf(logger_name=None):
     # Reload logging conf from env, then update the logging_conf
-    try:
-        del(conf['env'])
-    except KeyError:
-        # env not loaded yet
-        pass
+    # try:
```

Comment:
@kedark3 do you want to remove these lines?

Reply:
removing it now as I see no effect of this in test run on jenkins.